python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -1083,9 +1083,19 @@ def _replace_closurevars(
     # Added on a per-proxy basis
     # https://github.com/rapidsai/xdf/pull/306#pullrequestreview-1636155428
     # "__hash__",
+    "__iadd__",
+    "__iand__",
+    "__ifloordiv__",
+    "__imod__",
+    "__imul__",
     "__int__",
     "__invert__",
+    "__ior__",
+    "__ipow__",
+    "__isub__",
     "__iter__",
+    "__itruediv__",
+    "__ixor__",
```

Comment:
Is there a reason not to use every operator listed in https://docs.python.org/3/library/operator.html#in-place-operators? This is meant to be a generic proxy, so we should list the entire set of Python operators.

This was missing `__iconcat__, __ilshift__, __imatmul__, __irshift__`.

I know that `Series` implements `__imatmul__`:
```python
>>> s = pd.Series([1, 2, 3])
>>> s @= s
>>> s
14
```

I don't know of any `pandas` object that implements the other missed operators, but other proxied objects (NumPy arrays) implement the shift operators:
```python
>>> arr = np.array([1, 2, 3])
>>> arr <<= 2
>>> arr
array([ 4,  8, 12])
```

And then `__iconcat__` is just there for completeness. Weirdly, this operator _doesn't_ concatenate for `pd.Series` -- it adds:

```python
>>> s = pd.Series([1, 2, 3])
>>> operator.iconcat(s, s)
0    2
1    4
2    6
dtype: int64
```

...and it raises for NumPy arrays:

```python
>>> arr = np.array([1, 2, 3])
>>> operator.iconcat(arr, arr)
...
TypeError: Concatenation operation is not implemented for NumPy arrays, use np.concatenate() instead. Please do not rely on this error; it may not be given on all Python implementations.
```

```suggestion
    "__iadd__",
    "__iand__",
    "__iconcat__",
    "__ifloordiv__",
    "__ilshift__",
    "__imatmul__",
    "__imod__",
    "__imul__",
    "__int__",
    "__invert__",
    "__ior__",
    "__ipow__",
    "__irshift__",
    "__isub__",
    "__iter__",
    "__itruediv__",
    "__ixor__",
```